newIDE/app/src/InstancesEditor/SelectionRectangle.js

```diff
@@ -71,6 +109,41 @@ export default class SelectionRectangle {
         this._instancesInSelectionRectangle.push(instance);
       }
     };
+    this.temporarySelector = new gd.InitialInstanceJSFunctor();
+    // $FlowFixMe - invoke is not writable
+    this.temporarySelector.invoke = instancePtr => {
+      // $FlowFixMe - wrapPointer is not exposed
+      const instance = gd.wrapPointer(instancePtr, gd.InitialInstance);
+      const instanceAABB = this.instanceMeasurer.getInstanceAABB(
+        instance,
+        this._temporaryAABB
+      );
+
+      const {
+        temporarySelectionRectangleEnd,
+        temporarySelectionRectangleStart,
+      } = this;
+      if (!temporarySelectionRectangleStart || !temporarySelectionRectangleEnd)
+        return;
```

Comment:
Why this return? This means that even if these are not defined, we still run `this.instances.iterateOverInstances` on a temporarySelector that does nothing? This is a lot of useless iterations?

Reply:
I used the same body function for this functor than for the other one. I think this is mainly a Flow issue